cmd/argocd-repo-server/commands/argocd_repo_server.go

```diff
@@ -200,7 +200,7 @@ func NewCommand() *cobra.Command {
 		},
 	}
 	if cmdutil.LogFormat == "" {
-		cmdutil.LogFormat = os.Getenv("ARGOCD_REPO_SERVER_LOGLEVEL")
+		cmdutil.LogFormat = os.Getenv("ARGOCD_REPO_SERVER_LOGFORMAT")
 	}
```

Comment:
@ishitasequeira do you remember what these 3 lines were for from https://github.com/argoproj/argo-cd/commit/f533ed17f0cefef7cf86f5f27bcb0c1ee7fbeb36 ?

Looks to me like line 205 is always gonna overwrite this value.

Reply:
It's from a very long time. But looks like we might have missed catching that. I think we can completely remove lines  202-204, that is the entire if check.

Reply:
@chanhz would you mind updating the PR to delete all 3 lines?

Reply:
sure, thx for quick response.